### PR TITLE
feat: Send button actions

### PR DESCRIPTION
**Pull request by bennyn**
- http://localhost:21080/swagger-ui/#/Instance/sendButtonAction

### CODE REVIEW

File: src/InstanceService.ts [hunk 1]
```diff
@@ -397,6 +397,26 @@ export class InstanceService {
...
   }
 
+  async sendButtonAction(
+    instanceId: string,
+    conversationId: string,
+    referenceMessageId: string,
+    buttonId: string,
+  ): Promise<string> {
+    const instance = this.getInstance(instanceId);
+    const service = instance.account.service;
```
Comment (Yserz) [inline src/InstanceService.ts]: Is it possible that instance is undefined?

File: swagger.json [hunk 2]
```diff
@@ -1976,6 +1994,73 @@
...
+            }
+          },
+          "500": {
+            "description": "Internal server error",
+            "schema": {
+              "$ref": "#/definitions/ServerErrorMessage"
+            }
+          }
+        },
+        "summary": "Send a button action to a poll",
```
Comment (Yserz) [inline swagger.json]: I think the summaries are interchanged

File: src/InstanceService.ts (hunk 1 above)
Comment (bennyn) [reply src/InstanceService.ts]: At this stage not. Otherwise tsc would also fail to compile.

File: swagger.json (hunk 2 above)
Comment (bennyn) [reply swagger.json]: You are right. Will fix it.

File: src/InstanceService.ts (hunk 1 above)
Comment (Yserz) [reply src/InstanceService.ts]: Ah okay, it just looks like instanceId is coming from the outside. In other words it seems to be guaranteed that an instance exists for the given instanceId. Alright :)

**Pull request merged**